### PR TITLE
OPENEUROPA-3251: Initialize Open Vocabularies component.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "require": {
+        "drupal/core": "^8.7",
+        "php": ">=7.2"
+    },
+    "require-dev": {
+        "composer/installers": "~1.5",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "~9.0@stable",
```
Comment (nagyad) [inline composer.json]: "9.x-dev#93075ab"

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "guzzlehttp/guzzle": "~6.3",
+        "openeuropa/code-review": "~1.0.0-beta2",
```
Comment (nagyad) [inline composer.json]: "^1.3.0"

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        "openeuropa/drupal-core-require-dev": "^8.7",
```
Comment (nagyad) [inline composer.json]: "^8.8"

File: open_vocabularies.info.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+name: Open Vocabularies
+description: Open Vocabularies component.
```
Comment (nagyad) [inline open_vocabularies.info.yml]: Could we collect some meaningful description here?

File: open_vocabularies.module [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * @file
+ * Open Vocabularies module.
+ */
+
+declare(strict_types = 1);
```
Comment (nagyad) [inline open_vocabularies.module]: Remove this file, we will add it once we need it.

**Review by nagyad — CHANGES_REQUESTED** [review]

File: open_vocabularies.info.yml (hunk 4 above)
Comment (nagyad) [reply open_vocabularies.info.yml]: Use a sentence from the readme.

**Review by nagyad — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,102 @@
-# open_vocabularies
\ No newline at end of file
+# Open Vocabularies
+
+The Open Vocabularies component allows you to create “open” vocabularies. These vocabularies wrap categorisation systems, like Drupal taxonomies or Publication Office vocabularies, or subsets of them.
+It provides a field type that can be added to fieldable entities. Then associations between the vocabularies and the field instances can be created, allowing to categorise the content in a flexible way.
```
Comment (ademarco) [inline README.md]: Shall we try with something like this?

The Open Vocabularies module allows users to choose how a piece of content will be categorised without having to change the related content type field definition.
It does that by providing:

- An open vocabulary field type
- The possibility to create associations between open vocabulary field types and Drupal categorisation systems, such as Drupal taxonomies, Publication Office vocabularies, or subsets of them.

Once a content type is equipped with an open vocabulary field, users with appropriate access can decide which type of entities that field will able to reference, by creating associations.

File: composer.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
+{
+    "name": "openeuropa/open_vocabularies",
+    "description": "Open Vocabularies component.",
```
Comment (ademarco) [inline composer.json]: Let's use this one maybe:
```
Choose how a piece of content will be categorised without having to change the related content type field definition.
```

**Review by ademarco — CHANGES_REQUESTED** [review]

File: composer.json [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    "require": {
+        "drupal/core": "^8.7",
+        "php": ">=7.2"
+    },
+    "require-dev": {
+        "composer/installers": "~1.5",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "9.x-dev#93075ab",
```
Comment (ademarco) [inline composer.json]: We can now use ~9.0

File: open_vocabularies.info.yml [hunk 9]
```diff
@@ -0,0 +1,6 @@
+name: Open Vocabularies
+description: The Open Vocabularies component allows you to create “open” vocabularies that wrap categorisation systems, like Drupal taxonomies or Publication Office vocabularies, or subsets of them.
```
Comment (ademarco) [inline open_vocabularies.info.yml]: Let's reuse the same on composer.json:
```
Choose how a piece of content will be categorised without having to change the related content type field definition.
```

**Review by ademarco — CHANGES_REQUESTED** [review]

**Review by ademarco — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**